src/test/ui/closures/2229_closure_analysis/migrations/insignificant_drop_attr.rs

```diff
@@ -0,0 +1,58 @@
+// run-pass
+
+#![deny(disjoint_capture_drop_reorder)]
+#![feature(rustc_attrs)]
+#![allow(unused)]
+
+#[rustc_insignificant_dtor]
+struct InsignificantDropPoint {
+    x: i32,
+    y: i32,
+}
+
+impl Drop for InsignificantDropPoint {
+    fn drop(&mut self) {}
+}
+
+// `String` implements drop which makes `Person` a droppable type.
+struct Person {
+    age: i32,
+    name: String,
+}
+
+// `TwoPeople` is also droppable as `Person` is. But since it is
```

Comment:
TwoPeople is a Drop Type since it contains feilds that need drop.
/ since it contains person which is a Drop Type
...